### PR TITLE
add serious fuzz to the timer, but still testing for some accuracy

### PR DESCRIPTION
**Pull request by GantMan**
This test is failing too often.   I ran it a ton of times and saw the worst case scenario was always > 0.008, so I'm setting the test at that threshold.   This should take care of Travis Failures due to this evaluation.

### CODE REVIEW

**Comment by GantMan** [comment]
As you can see from https://travis-ci.org/infinitered/rmq/builds/54253461  my fuzzy wasn't very fuzzy... was he?

**Comment by GantMan** [comment]
This madness linked to #207

File: spec/app.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@
     before_time = Time.now
     @app.after 0.01 do
       after_time = Time.now
-      (after_time - before_time).should.be >= 0.01
+      (after_time - before_time).should.be >= 0.007
```
Comment (jamonholmgren) [inline spec/app.rb]: Maybe it's some sort of NSTimer inaccuracy at these small time periods.

Comment (GantMan) [inline spec/app.rb]: I'm assuming there's some allowed bit of error for short amounts of time.

**Pull request merged**